8.8/build.sh

```diff
@@ -15,9 +15,6 @@ fi
 
 deps="${1:-all}"
 
-# export this to vips.modules ... cmake needs it
-export BASEDIR=$basedir
```

Comment:
Removed, I've set `cmakeargs` and `mesonargs` in `jhbuildrc` instead:
https://github.com/libvips/build-win64/pull/26/files#diff-f876610988e7cf5e587ff77317650becR133

Reply:
We should update 8.8/README.md to remove the ref to this. It'll need to note that you need a recent jhbuild for meson support as well.

Reply:
Oh, there are loads of other things too, like pe-util. We should just point people to the Dockerfile for hints on setting up for a non-Docker build.